### PR TITLE
#70: Use `project-keeper`

### LINKED ISSUE #70: Use `project-keeper`
Like our other projects, we should use `project-keeper` to unify the layout and content.

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #70.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +21,10 @@ before_script:
   - echo '{}' > "$HOME"/.docker/config.json
 
 script:
-  - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent org.jacoco:jacoco-maven-plugin:prepare-agent-integration verify sonar:sonar -Dsonar.projectKey=com.exasol:row-level-security -Dsonar.login=${SONAR_TOKEN}
+  - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent org.jacoco:jacoco-maven-plugin:prepare-agent-integration
+    verify sonar:sonar -Dsonar.login=${SONAR_TOKEN}
+    -Dsonar.coverage.jacoco.xmlReportPaths=target/site/jacoco/jacoco.xml,target/site/jacoco-it/jacoco.xml
```
Comment (chiaradiamarcelo) [inline .travis.yml:28]: I think you need to add `-Dcom.exasol.dockerdb.image=${DOCKER_DB_VERSION}` as well if you wanna use the images defined above

File: .travis.yml [hunk 2]
```diff
@@ -4,9 +4,12 @@ language: java
...
 
-matrix:
-  include:
-    - jdk: openjdk11
+jdk:
+  - openjdk11
+
+env:
+  - DOCKER_DB_VERSION=7.0.2
+  - DOCKER_DB_VERSION=6.2.12
```
Comment (chiaradiamarcelo) [inline .travis.yml:12]: why these 2 env vars have the same name?

File: README.md [hunk 3]
```diff
@@ -62,6 +62,7 @@ Running the Virtual Schema requires a Java Runtime version 11 or later.
 | [Exasol Testcontainers](https://github.com/exasol/exasol-testcontainers)            | Integration test Exasol instance on Docker             | MIT License                   |
 | [Hamcrest Resultset Matcher](https://github.com/exasol/hamcrest-resultset-matcher)  | Validating JDBC resultsets                             | MIT License                   |
 | [Java Hamcrest](http://hamcrest.org/JavaHamcrest/)                                  | Checking for conditions in code via matchers           | BSD License                   |
+| [Jacoco Agent](https://www.jacoco.org/jacoco/)                                      | Collecting coverage information                        | Eclipse Public License 2.0    |
```
Comment (chiaradiamarcelo) [inline README.md:65]: should we also add a line for the `project-keeper` in the `Build Dependencies` section? as:

`| [Project Keeper Maven Plugin](https://github.com/exasol/project-keeper-maven-plugin)         | Checking project structure                             | MIT License                   |`

File: pom.xml [hunk 4]
```diff
@@ -139,12 +163,35 @@
                             <goal>report</goal>
                         </goals>
                     </execution>
+
```
Comment (chiaradiamarcelo) [inline pom.xml]:
```suggestion
```

**Review by chiaradiamarcelo — COMMENTED** [review]
looks good, left a few comments

File: .travis.yml (hunk 1 above)
Comment (redcatbear) [reply .travis.yml:28]: Done.

File: .travis.yml (hunk 2 above)
Comment (redcatbear) [reply .travis.yml:12]: Those are variants. Travis will permutate them.

File: README.md (hunk 3 above)
Comment (redcatbear) [reply README.md:65]: Fixed.

File: pom.xml (hunk 4 above)
Comment (redcatbear) [reply pom.xml]: Fixed.

**Review by chiaradiamarcelo — APPROVED** [review]

**Pull request merged**